pupil-api/readme.md

```diff
@@ -7,44 +7,6 @@ Required environment variables
 Optional environment variables
 `ENVIRONMENT_NAME` - name of the environment this API is running in.
 
-## Application healthcheck
-
-**Endpoint**: GET - http://localhost:3003/ping
-
-**Description**: Returns current server time.  
-Also inclues last commit and build deployment details if generated during build. 
-
-
-#### Header
-```
-{
-}
-```
-
-
-#### Body
-```
-```
-
-## Authentication and data collection
-
-**Endpoint**: POST - http://localhost:3003/auth
-
-**Description**: Used to authenticate a pupil logging in on the SPA and send back the SPA data required to take a check.
-
-#### Header
-```
-{
-	Content-Type: application/json
-}
-```
-
-
-#### Body
-```
-{
-	"pupilPin": "<some pin>",
-	"schoolPin": "<some pin>"
-}
-```
+## Swagger API Documentation
 
+When running in development the api documentation can be found at http://devserver:3003/api-docs
```

Comment:
`devserver` seems arbitrary.  Wouldn't `localhost` be better,  as it is more common,  here?

Reply:
agreed.  @danwilliams please change docs.

Reply:
I am unsure that I agree with changing it to localhost... I could note that localhost is a possibility, but it can critically be any hostname, and I think it's dangerous to rely too much on localhost as a default?

Reply:
`devserver` is a generic name to illustrate a concept. My dev machines are more interestingly-named 😄 The point of what I was saying is that localhost is not the only host to use, and should not really be the only host we work with or imply. (I don't think this is the place for me to go into all the reasons not to use localhost.) In any case, the change was made - I thought I'd just make this further note.

Reply:
point acknowledged :) 
i think it makes it obvious to the reader when it says `localhost`